### ISSUE TITLE
V0.11 requesting :cursor_scroll input not sending :cursor_scroll events to scene

### ISSUE THREAD

**Issue by vacarsu** [boydm/scenic#226]
In the init function I `request_input(scene, [:cursor_scroll])` and have a handle input to receive the message. However the handle_input never gets hit. I'm using the macbook trackpad, maybe it is not compatible with scenic?

```
def handle_input(
    {:cursor_scroll, {offset, position}},
    _,
    %{assigns: %{state: _state}} = scene
  ) do
    Logger.debug(inspect offset)
    Logger.debug(inspect position)

    {:noreply, scene}
end
```

**Comment by boydm** [comment]
This is fixed in the v0.11 branch.

This includes a few other input bugs/changes. The big one is that :cursor_pos and :cursor_scroll inputs are sent to all scenes that requested them, not just those who's :input style the cursor is over.

If you could verify the fix works for you , then close the issue, that would be appreciated.

**Comment by vacarsu** [comment]
Working now!
